articles/confidential-computing/faq.md

```diff
@@ -27,32 +27,35 @@ If your Azure issue is not addressed in this article, visit the Azure forums on
-    -	**InsufficientQuota**: [Create a support request to increase your quota](../azure-portal/supportability/per-vm-quota-requests.md). Free trial subscriptions don't have quota for confidential computing VMs. 
+    
+1. **I get an ARM deployment failure error "Operation could not be completed as it results in exceeding approved standard DCSV2 Family Cores Quota**
+
+-	**InsufficientQuota**: [Create a support request to increase your quota](../azure-portal/supportability/per-vm-quota-requests.md). Free trial subscriptions don't have quota for confidential computing VMs. 
 
 1. **DCsv2 virtual machines don't show up when I try to search for them in the portal size selector**
 
    Make sure you've selected an available region. Also make sure you select “clear all filters” in the size selector. 
 
 1. **What’s the difference between DCsv2-Series and DC-Series VMs?**
 
-   DC-Series VMs run on older 6-core Intel Processors with SGX. These have less total memory, less EPC (Enclave Page Cage) memory, and are available in less regions. These VMs are only available in US East and Europe West are available in two sizes: Standard_DC2s and Standard_DC4s. They will not go GA and can only be deployed in the [Confidential Compute DC-Series VM [Preview]](https://azuremarketplace.microsoft.com/marketplace/apps/microsoft-azure-compute.confidentialcompute?tab=Overview) Marketplace instance
+   DC-Series VMs run on older 6-core Intel Processors with SGX and have less memory, smaller EPC (Enclave Page Cage) memory, and available in less number of regions(US East and Europe West in Standard_DC2s and Standard_DC4s sizes). These VM's are not planned to be made Generally Available and not recommended. If you still like to deploy these VM's please use this Azure Market place template - [Confidential Compute DC-Series VM [Preview]](https://azuremarketplace.microsoft.com/marketplace/apps/microsoft-azure-compute.confidentialcompute?tab=Overview) Marketplace instance
 
 1. **Are DCsv2 virtual machines available globally?**
 
-   No, these virtual machines are only available in select regions. Check the [products by regions page](https://azure.microsoft.com/global-infrastructure/services/?products=virtual-machines) for the latest available regions. 
+   Not yet, these virtual machines are now available in select regions [in these select regions](https://azure.microsoft.com/global-infrastructure/services/?products=virtual-machines) for the latest available regions. 
 
-1. **How do I install the Open Enclave SDK?**
+1. **How do I install the Open Enclave SDK on the DCsv2 VM's?**
    
-   For instruction on how to install the OE SDK on a machine whether in Azure or on-premise, follow the instructions on the [Open Enclave SDK GitHub](https://github.com/openenclave/openenclave).
+   For instructions on how to install the OE SDK on an Azure or on-premise machines, follow the instructions on the [Open Enclave SDK GitHub](https://github.com/openenclave/openenclave).
```

Comment:
Either "on Azure or on-premise machines" or "on an Azure or on-premise machine"

Reply:
accepted. check grammer fixes may be